pkg/endpointslices/endpointslices.go

```diff
@@ -29,6 +29,15 @@ type EndpointSlicesExporter struct {
 	sliceInfo  []EndpointSlicesInfo
 }
 
+type NoOwnerRefErr struct {
```

Comment:
what do u think to add this to utils?
since its not EndpointSlice

Reply:
I think we can leave it here as we don't use it in any other place and I don't want to make utils a kitchen sink